lua/lspconfig/util.lua

```diff
@@ -222,6 +225,10 @@ function M.server_per_root_dir_manager(_make_config)
     local client_id = clients[root_dir]
     if not client_id then
       local new_config = _make_config(root_dir)
+      -- do nothing if the client is not enabled
+      if new_config.enabled == false then
```

Comment:
more lu-onic to do if not new_config.enabled imo

Reply:
`new_config.enabled` will normally not be set, so it will be `nil`. That's why a check for `false` is needed here

Reply:
ah, got it